### PR TITLE
Hide version number option

### PR DESCRIPTION
**Pull request by iwai-wovn**
- Added a new `showVersion` setting.

Set showVersion to false, you can hide the version number of the X-Wovn-Handler header.

Note: Since there was a request for java7_support, java7_support will be merged with master and java6_support in the same way.

### CODE REVIEW

File: src/main/java/com/github/wovnio/wovnjava/WovnServletFilter.java [hunk 1]
```diff
@@ -41,7 +41,11 @@ public void doFilter(ServletRequest request, ServletResponse response, FilterCha
         if (((HttpServletResponse)response).containsHeader("X-Wovn-Handler")) {
             isRequestAlreadyProcessed = true;
         } else {
-            ((HttpServletResponse)response).setHeader("X-Wovn-Handler", "wovnjava_" + Settings.VERSION);
+            if (this.settings.showVersion) {
```
Comment (bstewart00) [inline src/main/java/com/github/wovnio/wovnjava/WovnServletFilter.java]: I would make a method to generate the value:
```
    private String getWovnHandlerHeaderValue() {
        if (this.settings.showVersion) {
            return "wovnjava_" + Settings.VERSION;
        } else {
            return "wovnjava"
        }
    }
```

Also generate it inside `init` so we only do it once
`this.wovnHandlerHeaderValue = this.getWovnHandlerHeaderValue(settings)`

File: README.md [hunk 2]
```diff
@@ -336,6 +337,18 @@ This is intended to better understand what the problem is if something is not wo
 
 _Note that `wovnCacheDisable` and `wovnDebugMode` is only available when debugMode is turned on in your wovnjava configuration._
 
+### 2.12. showVersion
+
+A flag to disable hide the version number of `X-Wovn-Handler` header.
```
Comment (torkelbe) [inline README.md]:
```suggestion
A flag to control whether or not to show wovnjava version number in the `X-Wovn-Handler` HTTP response header.
```

File: README.md [hunk 3]
```diff
@@ -336,6 +337,18 @@ This is intended to better understand what the problem is if something is not wo
...
+
+Turn off showVersion by setting the parameter to false.
```
Comment (torkelbe) [inline README.md]:
```suggestion
Hide wovnjava version number from HTTP header by setting `showVersion` to false.
```

**Comment by bstewart00** [comment]
How was this merged without approvals? Oh, it's targeted a non-master branch. Please wait for approvals next time.

**Comment by iwai-wovn** [comment]
> How was this merged without approvals? Oh, it's targeted a non-master branch. Please wait for approvals next time.

Sorry, ok. I’m will wait approvals.

**Comment by bstewart00** [comment]
The changes were good anyway so no problem

**Comment by iwai-wovn** [comment]
I will be careful from next

2020年3月26日(木) 13:33 Brett Stewart <notifications@github.com>:

**Pull request merged**